tests/unit/controllers/events/view/edit/basic-details-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Definition for rule 'ember/no-get-properties' was not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4845812)